tests/lib/snaps/test-snapd-audio-record/Makefile

```
@@ -0,0 +1,8 @@
+all:
+	cd ./src && make all
```

The usual way to handle this is:
```suggestion
all install clean:
	$(MAKE) -C src $@
```